spec/latest/json-ld-api/index.html

```diff
@@ -448,9 +457,9 @@
       <dfn data-lt="flattened">flattening</dfn> goes a step further to ensure that the shape of the data
       is deterministic. In expanded documents, the properties of a single
       <a>node</a> may be spread across a number of different
-      <a>JSON objects</a>. By flattening a
+      <a class="changed">dictionaries</a>. By flattening a
```

Comment:
The algorithms operate on the internal representation, and this is a description of the algorithm.

I did give a thought to using "abstract representation" (or "abstract syntax"), but that is not as helpful for discussing a transformation. Danbri's "Infoset" idea is also similar.

The point here, though, is to separate the algorithms from the serialization.